rca/project_styleguide/templates/patterns/molecules/tab-nav/tab-nav-school.html

```diff
@@ -0,0 +1,12 @@
+<div class="tab-nav {{ modifier }} {% if sticky %}sticky{% endif %}" role="tablist" aria-label="School Tabs">
+    <div class="tab-nav__layout grid">
+    <nav class="tab-nav__container" role="tablist" aria-label="Tabs">
+        {# TODO switch from 'jump menu' to page.title #}
+        <span class="tab-item js-tab-item" role="tab" aria-selected="true">Jump to / {{ page.title }}</span>
+        {% for tab in tabs %}
+            <a id="tab-{{ tab.title|slugify }}" class="tab-item js-tab-item {% if forloop.counter == 1 %}active{% endif %}" href="#{{ tab.title|slugify }}" role="tab" aria-controls="{{ tab.title|slugify }}" data-tab="{{ tab.title|slugify }}" aria-selected="true">{{ tab.title }}</a>
```

Comment:
`|slugify` is being called four times, would it be worth using `with` to assign it to a varible?